ci/install_yq.sh

```diff
@@ -18,7 +18,9 @@ function install_yq() {
 	GOPATH=${GOPATH:-${HOME}/go}
 	local yq_path="${GOPATH}/bin/yq"
 	local yq_pkg="github.com/mikefarah/yq"
-	[ -x  "${GOPATH}/bin/yq" ] && return
+	local yq_version=3.4.1
+
+	[ -x  "${GOPATH}/bin/yq" ] && [ "`${GOPATH}/bin/yq --version`"X == "yq version ${yq_version}"X ] && return
```

Comment:
With this equality comparison will force install yq  of version `3.4.1` even other versions (maybe above 3.4.1) are installed. Using `[ "`${GOPATH}/bin/yq --version`" \> "yq version ${yq_version}" ]` will install yq only when the version is samller than 3.4.1.

```
$ [ "yq version 3.4.1" \> "yq version 2.x" ] && echo ">"
>
$ [ "yq version 3.4.1" \< "yq version 4" ] && echo "<"
<
```

Reply:
I tried
```
[ "`${GOPATH}/bin/yq --version`" <= "yq version ${yq_version}" ]
```
but got
```
./install_yq.sh
./install_yq.sh: line 25: [: yq version 3.4.1: unary operator expected
```

Could you help me with this issue?

Reply:
I worried that limit version to >= 3.4.1 will make yq command not work ok because commands of new version yq may be modified like yq 2 to yq 3.
What about just limit yq version to 3.4.1?